### PR TITLE
Example doc edits

### LINKED ISSUE #3411: Grammar / Style update for Examples

### PR DESCRIPTION
**Pull request by dwelsch-esi**
Edited notebook examples.

### CODE REVIEW

File: Docs/user_guide/examples.rst [hunk 1]
```diff
@@ -9,15 +9,22 @@ AIMET examples
...
+- Teach you how to use AIMET quantization and compression techniques
 
-For more details on each of the features and APIs see :ref:`Links to User Guide and API Documentation<ug-index>`.
+For a discussion of quantization techniques, see :doc:`/user_guide/model_quantization`.
+
+For a discussion of compression techniques, see :doc:`/user_guide/model_compression`.
+
+For the API reference, see:
+- :doc:`/api_docs/torch` for PyTorch
+- :doc:`/api_docs/keras` for TensorFlow Keras
```
Comment (quic-hitameht) [inline Docs/user_guide/examples.rst]: Minor: We should only say `TensorFlow`

File: Docs/user_guide/examples.rst [hunk 2]
```diff
@@ -73,40 +80,106 @@ Running the notebooks
...
+      WORKSPACE="<absolute_path_to_workspace>"
+
+2. Create and move to the workspace:
+
+   .. code-block:: shell
+    
+      mkdir $WORKSPACE && cd $WORKSPACE
+
+3. Identify the release tag (``<release_tag>``) of the AIMET package that you're working with at: https://github.com/quic/aimet/releases.
+<<<<<<< HEAD
```
Comment (quic-hitameht) [inline Docs/user_guide/examples.rst]: Could this marker have been unintentionally added while resolving a merge conflict?

File: Docs/user_guide/examples.rst [hunk 3]
```diff
@@ -73,40 +80,106 @@ Running the notebooks
...
-#. The command generates and displays a URL in the terminal. Copy and paste the URL into your browser.
-#. Install AIMET and its dependencies using the instructions in :doc:`install`.
+      jupyter notebook --ip=* --no-browser &
 
+   The command generates and displays a URL in the terminal. 
+   
+3. Copy and paste the URL into your browser.
 
-1. Download the example notebooks and related code
+4. Install AIMET and its dependencies using the instructions in :doc:`install`.
```
Comment (quic-hitameht) [inline Docs/user_guide/examples.rst]: Looks like this is not referencing to the right install package. I don't think we have rst named 'install'

File: Examples/torch/quantization/autoquant.ipynb [hunk 4]
```diff
@@ -10,39 +10,59 @@
...
     "2. Load a pretrained FP32 model\n",
     "3. Run AutoQuant\n",
     "\n",
-    "#### What this notebook is not\n",
-    "This notebook is not designed to show state-of-the-art AutoQuant results. For example, it uses a relatively quantization-friendly model like Resnet18. Also, some optimization parameters are deliberately chosen to have the notebook execute more quickly.\n"
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "\n",
+    "This notebook does not show state-of-the-art QAT results. For example, it uses a relatively quantization-friendly model (Resnet18). Also, some optimization parameters like number of fine-tuning epochs are chosen to improve execution speed in the notebook.\n",
```
Comment (quic-hitameht) [inline Examples/torch/quantization/autoquant.ipynb]: Copy - paste error, it shouldn't say state-of-the-art QAT results

File: Examples/torch/quantization/adaround.ipynb [hunk 5]
```diff
@@ -5,49 +5,63 @@
...
-    "#### What this notebook is not\n",
-    "* This notebook is not designed to show state-of-the-art results\n",
-    "* For example, it uses a relatively quantization-friendly model like Resnet18\n",
-    "* Also, some optimization parameters are deliberately chosen to have the notebook execute more quickly"
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "\n",
+    "This notebook does not show state-of-the-art QAT results. For example, it uses a relatively quantization-friendly model (Resnet18). Also, some optimization parameters like number of fine-tuning epochs are chosen to improve execution speed in the notebook.\n",
```
Comment (quic-hitameht) [inline Examples/torch/quantization/adaround.ipynb]: Copy - paste error, it shouldn't say state-of-the-art QAT results

File: Examples/torch/quantization/cle_bc.ipynb [hunk 6]
```diff
@@ -4,50 +4,74 @@
...
+    "3. Create a quantization simulation model (with fake quantization ops inserted) and evaluate the quantized simuation model\n",
+    "4. Apply CLE, BC, and high-bias folding and evaluate the simulation model\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
     "\n",
+    "Note\n",
     "\n",
-    "#### What this notebook is not\n",
-    "* This notebook is not designed to show state-of-the-art results. For example, it uses a relatively quantization-friendly model like Resnet18. Also, some optimization parameters are deliberately chosen to have the notebook execute more quickly.\n"
+    "This notebook does not show state-of-the-art QAT results. For example, it uses a relatively quantization-friendly model (Resnet18). Also, some optimization parameters like number of fine-tuning epochs are chosen to improve execution speed in the notebook.\n",
```
Comment (quic-hitameht) [inline Examples/torch/quantization/cle_bc.ipynb]: Copy - paste error, it shouldn't say state-of-the-art QAT results

File: Examples/torch/quantization/adaround.ipynb (hunk 5 above)
Comment (quic-hitameht) [reply Examples/torch/quantization/adaround.ipynb]: Let's just say state-of-the-art results

File: Examples/torch/compression/channel_pruning.ipynb [hunk 7]
```diff
@@ -4,48 +4,64 @@
...
+    "   2. Fine-tune the model\n",
     "\n",
     "\n",
-    "#### What this notebook is not \n",
-    "* This notebook is not designed to show state-of-the-art compression results. For example, some optimization parameters such as num_comp_ratio_candidates, num_eval_iterations and epochs are deliberately chosen to have the notebook execute more quickly.\n"
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
+    "\n",
+    "This notebook does not show state-of-the-art QAT results. For example, it uses a relatively quantization-friendly model (Resnet18). Also, some optimization parameters like number of fine-tuning epochs are chosen to improve execution speed in the notebook.\n",
```
Comment (quic-hitameht) [inline Examples/torch/compression/channel_pruning.ipynb]: Sam comment as above.

Comment (quic-hitameht) [inline Examples/torch/compression/channel_pruning.ipynb]: Same comment as above.

File: Examples/torch/compression/spatial_svd.ipynb [hunk 8]
```diff
@@ -4,48 +4,64 @@
...
-    "1. **Spatial SVD**: This is a tensor-decomposition technique generally applied to convolutional layers (Conv2D). Applying this technique will decompose a single convolutional layer into two. The weight tensor of the layer to be split is flattended to a 2D matrix and singular value decomposition (SVD) is applied to this matrix. Compression is achieved by discarding the least significant singular values in the diagonal matrix. The decomposed matrices are combined back into two separate convolutional layers.\n",
-    "2. **Channel Pruning**: In this technique AIMET will discard least significant (using a magnitude metric) input channels of a given convolutional (Conv2D) layer. The layers of the model feeding into this convolutional layer also have the channels dimension modified to get back to a working graph. This technique also uses a layer-by-layer reconstruction procedure that modifies the weights of the compressed layers to minimize the distance of the compressed layer output to the corresponding layer output of the original model.\n",
+    "See [AIMET spatial SVD](/user_guide/spatial_svd) for a detailed discussion of spatial SVD and other compression techniques.\n",
```
Comment (quic-hitameht) [inline Examples/torch/compression/spatial_svd.ipynb:13]: The link leads to a 404 error page.

File: Examples/torch/compression/channel_pruning.ipynb [hunk 9]
```diff
@@ -4,48 +4,64 @@
...
-    "2. **Channel Pruning**: In this technique AIMET will discard least significant (using a magnitude metric) input channels of a given convolutional (Conv2D) layer. The layers of the model feeding into this convolutional layer also have the channels dimension modified to get back to a working graph. This technique also uses a layer-by-layer reconstruction procedure that modifies the weights of the compressed layers to minimize the distance of the compressed layer output to the corresponding layer output of the original model.\n",
+    "See [Model Compression](/user_guide/model_compression) for a detailed discussion of channel pruning and other compression techniques.\n",
```
Comment (quic-hitameht) [inline Examples/torch/compression/channel_pruning.ipynb:13]: The link leads to a 404 error page.

File: Examples/torch/compression/spatial_svd_channel_pruning.ipynb [hunk 10]
```diff
@@ -4,50 +4,67 @@
...
-    "2. **Channel Pruning**: In this technique AIMET will discard least significant (using a magnitude metric) input channels of a given convolutional (Conv2D) layer. The layers of the model feeding into this convolutional layer also have the channels dimension modified to get back to a working graph. This technique also uses a layer-by-layer reconstruction procedure that modifies the weights of the compressed layers to minimize the distance of the compressed layer output to the corresponding layer output of the original model.\n",
+    "See [AIMET spatial SVD](/user_guide/spatial_svd_channel_pruning) for a detailed discussion of these and other compression techniques.\n",
```
Comment (quic-hitameht) [inline Examples/torch/compression/spatial_svd_channel_pruning.ipynb:13]: The link leads to a 404 error page.

File: Examples/onnx/quantization/cle.ipynb [hunk 11]
```diff
@@ -2,88 +2,94 @@
...
     "3. Create a quantization simulation model (with fake quantization ops inserted) and evaluate this simuation model to get a quantized accuracy score\n",
     "4. Apply CLE and evaluate the simulation model to get a post-finetuned quantized accuracy score\n",
     "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Note\n",
     "\n",
-    "#### What this notebook is not\n",
-    "* This notebook is not designed to show state-of-the-art results. For example, it uses a relatively quantization-friendly model like Resnet18. Also, some optimization parameters are deliberately chosen to have the notebook execute more quickly.\n"
+    "This notebook does not show state-of-the-art QAT results. For example, it uses a relatively quantization-friendly model (Resnet18). Also, some optimization parameters like number of fine-tuning epochs are chosen to improve execution speed in the notebook.\n",
```
Comment (quic-hitameht) [inline Examples/onnx/quantization/cle.ipynb]: Same copy-paste error.

File: Examples/onnx/quantization/quantsim.ipynb [hunk 12]
```diff
@@ -2,50 +2,64 @@
...
-   "metadata": {
-    "collapsed": false,
-    "jupyter": {
-     "outputs_hidden": false
-    }
-   },
+   "metadata": {},
    "source": [
-    "# Quantization Simulation\n",
+    "# Quantization-aware training\n",
```
Comment (quic-hitameht) [inline Examples/onnx/quantization/quantsim.ipynb:7]: We shouldn't call it `Quantization-aware training`. Keep `Quantization simulation`. We don't have QAT techniques in aimet-onnx

File: Examples/onnx/quantization/quantsim.ipynb [hunk 13]
```diff
@@ -2,50 +2,64 @@
...
     "\n",
-    "This notebook shows a working code example of how to use AIMET to perform quantization simulation (quantsim). Quantsim is an AIMET feature that adds quantization simulation ops (also called fake quantization ops sometimes) to a trained ML model in order to compute quantization encodings and estimate the resulting accuracy of the model when deployed on quantized ML accelerators.\n",
+    "This notebook contains a working example of AIMET Quantization-aware training (QAT). \n",
```
Comment (quic-hitameht) [inline Examples/onnx/quantization/quantsim.ipynb:9]: We shouldn't be mentioning QAT here

File: Examples/onnx/quantization/quantsim.ipynb [hunk 14]
```diff
@@ -2,50 +2,64 @@
...
+    "QAT is an AIMET feature that adds quantization simulation operations (also called fake quantization ops) to a trained ML model. \n",
```
Comment (quic-hitameht) [inline Examples/onnx/quantization/quantsim.ipynb:10]: Same comment as above

File: Examples/tensorflow/quantization/keras/qat_range_learning.ipynb [hunk 15]
```diff
@@ -202,25 +159,31 @@
...
+    "Before calculating the simulated quantized accuracy using QuantizationSimModel, fold the BatchNormalization (BN) layers into adjacent Convolutional layers. The BN layers that cannot be folded are left as they are.\n",
+    "\n",
+    "BN folding improves inference performance on quantized runtimes but can degrade accuracy on these platforms. This step simulates this on-target drop in accuracy. \n",
     "\n",
     "The following code calls AIMET to fold the BN layers of a given model. </br>\n",
-    "**NOTE: During folding, a new model is returned. Please use the returned model for the rest of the pipeline.**"
+    "**NOTE: During folding, a new model is returned. Please use the returned model for the rest of the pipeline.**\n",
```
Comment (quic-hitameht) [inline Examples/tensorflow/quantization/keras/qat_range_learning.ipynb:176]: Looks like you forgot to delete the NOTE here..

**Review by quic-hitameht — CHANGES_REQUESTED** [review]
Please address the requested changes.

File: Docs/user_guide/examples.rst [hunk 16]
```diff
@@ -73,40 +80,106 @@ Running the notebooks
...
+   https://www.image-net.org/download.php
+
+
+3. Run the notebooks
+--------------------
+
+1. Navigate to one of the following paths under the Examples directory and launch your 
+   chosen Jupyter Notebook (`.ipynb` extension):
+
+   - `Examples/torch/quantization/`
```
Comment (dwelsch-esi) [inline Docs/user_guide/examples.rst]: Cut/paste error when resolving merge conflict. Fixed.

File: Docs/user_guide/examples.rst (hunk 2 above)
Comment (dwelsch-esi) [reply Docs/user_guide/examples.rst]: Yes. See below.

**Pull request closed without merging**